sklearn/metrics/tests/test_score_objects.py

```diff
@@ -18,12 +19,15 @@
 from sklearn.metrics import (f1_score, r2_score, roc_auc_score, fbeta_score,
                              log_loss, precision_score, recall_score)
 from sklearn.metrics.cluster import adjusted_rand_score
-from sklearn.metrics.scorer import (check_scoring, _PredictScorer,
-                                    _passthrough_scorer)
+from sklearn.metrics.scorer import check_scoring
+from sklearn.metrics.scorer import _PredictScorer
+from sklearn.metrics.scorer import _PassthroughScorer
+from sklearn.metrics.scorer import _get_scorer_name
```

Comment:
cosmetic: If you are changing this import format, then you should change it everywhere